plugins/polio/js/src/domains/Campaigns/SubActivities/hooks/useSubActivityValidation.tsx

```diff
@@ -67,6 +70,34 @@ yup.addMethod(
     },
 );
 
+yup.addMethod(
+    yup.date,
+    'validateLqasImEndDate',
+    function validateLqasImEndDate(formatMessage) {
```

Comment:
Arrow function

Reply:
Not in yup custom methods, because we need `this`